modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/CalibrationQueues.scala

```diff
@@ -41,7 +41,7 @@ object CalQueueState {
 @Lenses
 final case class CalibrationQueues(queues: SortedMap[QueueId, CalQueueState]) {
   val queueTables: SortedMap[QueueId, TableState[CalQueueTable.TableColumn]] =
-    queues.mapValues(_.tableState)
+    queues.view.mapValues(_.tableState).to(SortedMap)
```

Comment:
I vaguely recall you can also do `.to[SortedMap]` is there any benefit on doing so?

Reply:
That was the case in 2.12, but in 2.13 it's replaced by `.to(SortedMap)` (or `.to(Collection)`).